Notebook s2 ndci

caitlinadams opened the pull request:
_Note: checklist below only relevant if committing to the [`develop`](https://github.com/GeoscienceAustralia/dea-notebooks/tree/develop) branch:_

### Proposed changes
New real world example notebook about Chlorophyll in waterbodies

### Checklist (replace `[ ]` with `[x]` to check off)
- [X] Notebook created using the [DEA-notebooks template](https://github.com/GeoscienceAustralia/dea-notebooks/tree/develop)
- [X] Remove any unused Python packages from `Load packages`
- [X] Remove any unused/empty code cells
- [x] Remove any guidance cells (e.g. `General advice`)
- [x] Ensure that all code cells follow the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/) for code. The `jupyterlab_code_formatter` tool can be used to format code cells to a consistent style: select each code cell, then click `Edit` and then one of the `Apply X Formatter` options (`YAPF` or `Black` are recommended).
- [X] Include relevant tags in the final notebook cell (refer to the [DEA Tags Index](https://docs.dea.ga.gov.au/genindex.html), and re-use tags if possible)
- [x] Clear all outputs, run notebook from start to finish, and save the notebook in the state where all cells have been sequentially evaluated
- [ ] Test notebook on both the `NCI` and `DEA Sandbox` (flag if not working as part of PR and ask for help to solve if needed)
- [ ] If applicable, update the `Notebook currently compatible with the NCI|DEA Sandbox environment only` line below the notebook title to reflect the environments the notebook is compatible with

robbibt commented:
@caitlinadams I've made some minor changes above: I rearranged the introduction sections to follow the template a bit better (e.g. moved 'The problem' into 'Background`, and added a 'Description' heading); have a look and see if you're OK with this! I've also made the double quotations consistent, and changed the print statements for the NDCI and NDWI time series to plots so it's a bit easier to interpret. Other than that, I'm just waiting on the NCI to come back online at 2pm today to do a final test of whether it works on the NCI too - will close this off once I do.

caitlinadams commented:
Thanks for the changes, @robbibt -- great idea to plot the data rather than just display the xarray. I've added some axes labels and changed the plot aspects so all three data plots are visually similar. The setup at the start looks good too, I'll make sure I change the order in the forestry notebook as well. 

I've also fixed the tags as per the new wiki. Would you mind double-checking to see if I missed anything obvious?
